### ISSUE TITLE
Long tail: Ensuring that there is a min payout

### ISSUE THREAD

**Issue by jeffdomke** [blockstack/app-mining#25]
First step is just to compare proposals and decide.

**Comment by hstove** [comment]
We have some thoughts about how to do this, but we'll announce this before a new round to give further incentive to developers.

This goes with a conversation around a different payouts model, which may be some kind of 'tiered' payout system. Top tier gets top 20%, next one gets 20% of remaining, etc.

**Comment by pstan26** [comment]
Yep, reached out to Dan Arieli's people and the Game Theorists who created the aggregation algorithm to help create proposals. Currently 1) things feel too zero sum and competitive with neighboring ranked apps, which could be bad for the community, and 2) folks outside of the top 24 get paid so little for their effort.

 Initial thinking is creating tiered payouts (tier 1 all gets paid the same amount, and so on for other tiers) rather than direct ranking payouts.

Currently, for each App Mining cohort, there is a determined “pot” of total earnings that will get paid to apps. App Mining began paying $100,000 USD each month, with earnings paid out in Bitcoin. The top app gets paid 20% of the total pot. So, for a pot of $100k, the top app receives $20,000 USD. The next app gets paid 20% of the remaining pot. The remaining pot is $80k, and 20% of that is $16,000. This process continues until every app has been paid.

Here is a chart that visualizes the decay in rewards, depending on rank:
![image](https://user-images.githubusercontent.com/5605552/52869514-cf165e00-3113-11e9-8994-d6c8311a99ee.png)

You can see that the payouts descend too quickly for large participation. In essence if you are getting paid single digits, the game feels "over" for you. Having a floor like $100 and then a cutoff may make more sense.

**Comment by jeffdomke** [comment]
Update: Game theorists are working on it.

**Comment by pstan26** [comment]
Yep confirmed they are still working on it. Hoping for something for this round, but can't promise anything at the moment.

**Comment by GinaAbrams** [comment]
This is still a priority for us to do thoughtfully, but the truth of the matter is that we'll have a lot more leeway to do this if payouts are more significant (e.g. $300k) down the line, subject to regulatory compliance. For now, I’d propose we pay the top 100 apps a minimum of $100 as a simple way to acknowledge the minimum amount of time spent on these apps in the earliest of days in App Mining.

**Comment by jeffdomke** [comment]
I think I would support flattening out the distribution, especially as we increase rewards in the future, so that apps # 2, # 3, etc all earn more. There would be less of a steep drop-off. But the base assumption that all apps should earn a minimum seems unproven to me. Why?

**Comment by friedger** [comment]
App publishers who received 0€ are less likely to spend more time on the development of the app, to invest in activities outside their comfort zone.

Personally, I can imagine to spent award money on advertising, on design, on gitcoin.co,..

**Comment by agnelvishal** [comment]
Dmail gets 20,000$ but an app with rank 50 gets only 0.5$. Felt the variation is too much.
Maybe apps with lower rank should be paid well too to encourage new apps.

**Comment by friedger** [comment]
The payouts are increasing since August. See also for details : https://docs.blockstack.org/develop/mining-rewards.html

**Comment by agnelvishal** [comment]
> The payouts are increasing since August. See also for details : https://docs.blockstack.org/develop/mining-rewards.html

Payouts are increasing. But will lower ranked apps get decent pay? Will it not be better if news apps are encouraged?

**Comment by friedger** [comment]
Top 10 new apps get $1000 each

**Comment by friedger** [comment]
@agnelvishal You can see the payouts here as well: https://docs.google.com/spreadsheets/d/1xQrGclubYJzYHVqUYxZqXXtkRTYQSrEJ1udA67OF0oY/edit#gid=0

By next year Top 50 apps will get $5000+

**Comment by friedger** [comment]
In November all apps received more than 10$.
Should be closed.
